### PR TITLE
Mina

### PR DESCRIPTION
**Pull request by minimanimox**
Exchange all absolute position to flex and grid

### CODE REVIEW

File: css/desktop.css [hunk 1]
```diff
@@ -73,25 +71,32 @@ header {
...
+      margin: 0 auto;
 
-      .inner-article {
-        img {
-          margin-top: 80px;
-          margin-right: 50px;
-          width: 460px;
-          float: left;
-        }
+      font-size: 15pt;
```
Comment (juniahn-dev) [inline css/desktop.css:80]: Why you mixed 'pt, px' ?

File: css/desktop.css [hunk 2]
```diff
@@ -111,53 +116,66 @@ header {
...
-      column-gap: 20px;
-
-      .link-container {
+      grid-template-columns: repeat(4, 270px);
+      grid-template-rows: repeat(2, 320px);
+      row-gap: 50px;
+      column-gap: 30px;
+      justify-content: center;
+
+      .link-container-1,
```
Comment (juniahn-dev) [inline css/desktop.css]: Make common css. Class name can be duplicatated.
`.link-container { background-repeat: no-repeat; background-size: cover; position: relative; }`

File: index.html [hunk 3]
```diff
@@ -1,344 +1,267 @@
...
-                  <img src="./assets/Arrow.png" alt="white-arrow" />
-                </p>
-              </div>
+      </div>
+    </section>
+    <section class="section3-container">
+      <h2>Our <span>Services</span></h2>
+      <div class="wrapper">
+        <div class="link-wrapper">
+          <div class="link-container-1">
```
Comment (juniahn-dev) [inline index.html]: Like this
`<div class="link-container link-container-1">`

Comment (minimanimox) [reply index.html]: I changed them as you said thanks

File: css/desktop.css (hunk 1 above)
Comment (minimanimox) [reply css/desktop.css:80]: I thought using pt for font-size seemed more specific, am I wrong?

Comment (juniahn-dev) [reply css/desktop.css:80]: There is no right answer but I prefer uniformity.
I just curious abount that so you do not fix it

**Review by juniahn-dev — APPROVED** [review]

**Pull request merged**